### PR TITLE
Added recipe for sympow

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

File: recipes/sympow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  build:
+    - toolchain
+    - pari
+  run:
+    # new_data invokes PARI's GP
+    - pari
+
+test:
+  commands:
+    - test -x ${PREFIX}/bin/sympow
```
Comment (isuruf) [inline recipes/sympow/meta.yaml]: Instead of this, do `sympow -help`

File: recipes/sympow/sympow.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+SYMPOW_LOCAL="$HOME"/.local/share/sympow
+
+if [ ! -d "${SYMPOW_LOCAL}" ]; then
+    mkdir -p "${SYMPOW_LOCAL}"
+    cp -r "${CONDA_PREFIX}"/usr/share/sympow/* "${SYMPOW_LOCAL}"
```
Comment (isuruf) [inline recipes/sympow/sympow.sh]: Is `${CONDA_PREFIX}"/usr/share/sympow/` correct ?

File: recipes/sympow/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+about:
+  home: http://sagemath.org/spkg/upstream/sympow/index.html
```
Comment (isuruf) [inline recipes/sympow/meta.yaml]: This link is not working

File: recipes/sympow/meta.yaml (hunk 1 above)
Comment (saraedum) [inline recipes/sympow/meta.yaml]: Surprising switch. I did not know.

File: recipes/sympow/sympow.sh (hunk 2 above)
Comment (saraedum) [inline recipes/sympow/sympow.sh]: Fixed.

File: recipes/sympow/meta.yaml (hunk 3 above)
Comment (saraedum) [inline recipes/sympow/meta.yaml]: Fixed, sorry.

**Comment by isuruf** [comment]
@ocefpaf, ready for review

**Comment by isuruf** [comment]
@ocefpaf, thanks.

@saraedum, welcome to conda-forge. https://github.com/conda-forge/sympow-feedstock has been created. You will be asked if you would like to join conda-forge within the next day. If you accept, you'll be given commit privileges for the repo and access to the CI. One important thing to note is that if you want to make a change, send a PR from a fork. Branches of the main repo are used for maintaining different versions only.

**Comment by ocefpaf** [comment]
No problem. 

Welcome @saraedum!

**Pull request merged**